android/beagle/src/main/java/br/com/zup/beagle/android/view/BeagleFragment.kt

```diff
@@ -53,6 +56,21 @@ internal class BeagleFragment : Fragment() {
         private const val JSON_SCREEN_KEY = "JSON_SCREEN_KEY"
     }
 
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        if (context is OnFragmentCallback) {
+            mListener = context
+        } else {
+            throw ClassCastException(context.toString()
+                .toString() + " must implement OnFragmentCallback")
```

Comment:
Hi! It`s seems like a second uncessary conversion .toString()

```suggestion
            throw ClassCastException(context.toString() +" must implement OnFragmentCallback")
```